app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -1,35 +1,61 @@
 package org.oppia.android.app.databinding;
 
+import android.annotation.SuppressLint;
 import android.view.View;
 import android.view.ViewGroup.MarginLayoutParams;
+
 import androidx.annotation.NonNull;
+import androidx.core.view.ViewCompat;
 import androidx.databinding.BindingAdapter;
 
-/** Holds all custom binding adapters that set margin values. */
+/**
+ * Holds all custom binding adapters that set margin values.
+ */
 public final class MarginBindingAdapters {
 
-  /** Used to set a margin-start for views. */
+  @SuppressLint("WrongConstant")
+  private static boolean isRtlLayout(View view) {
```

Comment:
Nit: please move to be below the ``public`` methods.

Reply:
Done.